packages/stream-model-handler/src/model-handler.ts

```diff
@@ -97,27 +97,10 @@ export class ModelHandler implements StreamHandler<Model> {
   async _applyGenesis(commitData: CommitData, context: Context): Promise<StreamState> {
     const payload = commitData.commit
     const isSigned = StreamUtils.isSignedCommitData(commitData)
-    if (!isSigned) {
-      throw Error('Model genesis commit must be signed')
+    if (isSigned) {
```

Comment:
Does that mean attempting to deploy existing signed models would fail?

Reply:
Yes, I think it would be weird if our example model is signed when all other models are not?

Reply:
Yes should be easy to make the change, just need to make sure we update the composite used in the docs before release.

Reply:
Ok, thanks :)